# translation of sk.po to
# Slovak translation for update-manager
# Copyright (C) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the update-manager package.
#
# Peter Chabada <sk-i18n_chabada_sk>, 2006.
# Jozef Bucha <bucha_jozef@centrum.sk>, 2007.
# Ivan Masár <helix84@centrum.sk>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: sk\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2012-06-14 00:53+0100\n"
"PO-Revision-Date: 2012-04-04 21:02+0000\n"
"Last-Translator: Pavol Klačanský <pavol@klacansky.com>\n"
"Language-Team: <en@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n==1) ? 1 : (n>=2 && n<=4) ? 2 : 0;\n"
"X-Launchpad-Export-Date: 2012-04-17 13:41+0000\n"
"X-Generator: Launchpad (build 15099)\n"

#. TRANSLATORS: download size of small updates, e.g. "250 kB"
#: ../DistUpgrade/utils.py:433 ../UpdateManager/Core/utils.py:433
#, python-format
msgid "%(size).0f kB"
msgid_plural "%(size).0f kB"
msgstr[0] "%(size).0f kB"
msgstr[1] "%(size).0f kB"
msgstr[2] "%(size).0f kB"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:436 ../UpdateManager/Core/utils.py:436
#, python-format
msgid "%.1f MB"
msgstr "%.1f MB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:206 ../DistUpgrade/distro.py:436
#, python-format
msgid "Server for %s"
msgstr "Server pre %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:224 ../DistUpgrade/distro.py:230
#: ../DistUpgrade/distro.py:246
msgid "Main server"
msgstr "Hlavný server"

#: ../DistUpgrade/distro.py:250
msgid "Custom servers"
msgstr "Vlastné servery"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:142
msgid "Could not calculate sources.list entry"
msgstr "Nemožno vypočítať položku v sources.list"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:251
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Nemožno lokalizovať žiadne balíčky súborov. Pravdepodobne toto nie je CD/DVD "
"Ubuntu alebo je určené pre inú architektúru."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:294
msgid "Failed to add the CD"
msgstr "Zlyhalo pridanie CD"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:295
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Pri pridávaní CD sa vyskytla chyba, prechod na vyššiu verziu bude prerušený. "
"Nahláste túto chybu, ak používate platné Ubuntu CD.\n"
"\n"
"Chybová správa bola:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeCache.py:151
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Odstrániť balíky v zlom stave"
msgstr[1] "Odstrániť balík v zlom stave"
msgstr[2] "Odstrániť balíky v zlom stave"

#: ../DistUpgrade/DistUpgradeCache.py:154
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Balíky „%s“ sú v nekonzistentnom stave a je potrebné ich preinštalovať, ale "
"neboli pre ne nájdené žiadne archívy. Chcete tieto balíky teraz odstrániť, "
"aby bolo možné pokračovať?"
msgstr[1] ""
"Balík „%s“ je v nekonzistentnom stave a je potrebné ich preinštalovať, ale "
"neboli preň nájdené žiadne archívy. Chcete tento balík teraz odstrániť, aby "
"bolo možné pokračovať?"
msgstr[2] ""
"Balíky „%s“ sú v nekonzistentnom stave a je potrebné ich preinštalovať, ale "
"neboli pre ne nájdené žiadne archívy. Chcete tieto balíky teraz odstrániť, "
"aby bolo možné pokračovať?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:255
msgid "The server may be overloaded"
msgstr "Server môže byť preťažený"

#: ../DistUpgrade/DistUpgradeCache.py:368
msgid "Broken packages"
msgstr "Poškodené balíky"

#: ../DistUpgrade/DistUpgradeCache.py:369
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Váš systém obsahuje poškodené balíky, ktoré nemôžu byť týmto programom "
"opravené. Pred pokračovaním ich opravte programom synaptic alebo apt-get."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:693
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Počas prípravy na aktualizáciu sa vyskytol neriešiteľný problém:\n"
"%s\n"
"\n"
"Tento problém mohlo spôsobiť to, že:\n"
"* aktualizujete na ešte nevydanú testovaciu verziu Ubuntu\n"
"* práve používate ešte nevydanú testovaciu verziu Ubuntu\n"
"* používate neoficiálne balíky softvéru, ktoré neposkytuje tím Ubuntu\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:703
msgid "This is most likely a transient problem, please try again later."
msgstr "Toto je pravdepodobne dočasný problém, prosím skúste to neskôr."

#: ../DistUpgrade/DistUpgradeCache.py:706
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Ak nič z tohto nie je relevantné, prosím, pošlite toto hlásenie o chybe "
"príkazom „ubuntu-bug update-manager“ v termináli."

#: ../DistUpgrade/DistUpgradeCache.py:711
#: ../UpdateManager/UpdateManager.py:1031
msgid "Could not calculate the upgrade"
msgstr "Nepodarilo sa vypočítať aktualizáciu"

#: ../DistUpgrade/DistUpgradeCache.py:762
msgid "Error authenticating some packages"
msgstr "Chyba pri overovaní niektorých balíkov"

#: ../DistUpgrade/DistUpgradeCache.py:763
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Nebolo možné overiť niektoré balíky. Príčinou mohol byť dočasný problém so "
"sieťou. Môžete to opäť skúsiť neskôr. Nižšie je uvedený zoznam neoverených "
"balíkov."

#: ../DistUpgrade/DistUpgradeCache.py:783
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Balík „%s“ je označený na odstránenie ale nachádza sa na zozname balíkov, "
"ktoré sa nemajú odstraňovať."

#: ../DistUpgrade/DistUpgradeCache.py:787
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Nevyhnutný balík „%s“ je označený na odstránenie."

#: ../DistUpgrade/DistUpgradeCache.py:796
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Pokus o nainštalovanie verzie „%s“ z čiernej listiny"

#: ../DistUpgrade/DistUpgradeCache.py:914
#, python-format
msgid "Can't install '%s'"
msgstr "Nie je možné nainštalovať „%s“"

#: ../DistUpgrade/DistUpgradeCache.py:915
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Nebolo možné nainštalovať požadovaný balík. Prosím, nahláste to ako chybu "
"príkazom „ubuntu-bug update-manager“ v termináli."

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:926
msgid "Can't guess meta-package"
msgstr "Nie je možné odhadnúť meta-balík"

#: ../DistUpgrade/DistUpgradeCache.py:927
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Váš systém neobsahuje žiaden z balíkov ubuntu-desktop, kubuntu-desktop, "
"xubuntu-desktop alebo edubuntu-desktop a nebolo možné zistiť, ktorú verziu "
"Ubuntu používate.\n"
" Prosím, než budete pokračovať, najprv nainštalujte jeden z vyššie uvedených "
"balíkov pomocou Synaptic alebo apt-get."

#: ../DistUpgrade/DistUpgradeController.py:114
msgid "Reading cache"
msgstr "Číta sa vyrovnávacia pamäť"

#: ../DistUpgrade/DistUpgradeController.py:223
msgid "Unable to get exclusive lock"
msgstr "Nepodarilo sa uzamknúť databázu softvéru"

#: ../DistUpgrade/DistUpgradeController.py:224
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Zvyčajne to znamená, že je už spustená iná aplikácia na správu balíkov (ako "
"apt-get alebo aptitude). Prosím, najskôr ukončite danú aplikáciu."

#: ../DistUpgrade/DistUpgradeController.py:257
msgid "Upgrading over remote connection not supported"
msgstr "Aktualizácia prostredníctvom vzdialeného pripojenia nie je podporovaná"

#: ../DistUpgrade/DistUpgradeController.py:258
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Spúšťate aktualizáciu prostredníctvom vzdialeného pripojenia ssh s "
"rozhraním, ktoré to nepodporuje. Prosím, skúste aktualizáciu v textovom "
"režime pomocou „do-release-upgrade“.\n"
"\n"
"Táto aktualizácia sa teraz preruší. Prosím, skúste to bez použitia ssh."

#: ../DistUpgrade/DistUpgradeController.py:272
msgid "Continue running under SSH?"
msgstr "Pokračovať v spojení cez SSH?"

#: ../DistUpgrade/DistUpgradeController.py:273
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Zdá sa, že táto relácia beží pod ssh. Neodporúča sa vykonávať aktualizáciu "
"prostredníctvom ssh, pretože v prípade poruchy je ťažšie ju opraviť.\n"
"\n"
"Ak chcete pokračovať, na porte „%s“ sa spustí ďalší ssh démon.\n"
"Chcete pokračovať?"

#: ../DistUpgrade/DistUpgradeController.py:287
msgid "Starting additional sshd"
msgstr "Štartuje sa ďalší sshd"

#: ../DistUpgrade/DistUpgradeController.py:288
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Aby sa uľahčilo obnovenie systému v prípade poruchy, spustí sa ďalší sshd na "
"porte „%s“. Ak nastane problém s momentálne bežiacim ssh môžete sa stále "
"pripojiť k tomuto ďalšiemu.\n"

#: ../DistUpgrade/DistUpgradeController.py:296
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Ak používate firewall, možno budete musieť dočasne otvoriť tento port. "
"Pretože je to potenciálne nebezpečná operácia, nevykoná sa automaticky. Port "
"môžete otvoriť napr. pomocou:\n"
"„%s“"

#: ../DistUpgrade/DistUpgradeController.py:368
#: ../DistUpgrade/DistUpgradeController.py:413
msgid "Can not upgrade"
msgstr "Nie je možné vykonať prechod na novšiu verziu"

#: ../DistUpgrade/DistUpgradeController.py:369
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Tento nástroj nepodporuje aktualizáciu z '%s' na '%s'."

#: ../DistUpgrade/DistUpgradeController.py:378
msgid "Sandbox setup failed"
msgstr "Zlyhalo vytvorenie pieskoviska"

#: ../DistUpgrade/DistUpgradeController.py:379
msgid "It was not possible to create the sandbox environment."
msgstr "Nebolo možné vytvoriť prostredie pieskoviska."

#: ../DistUpgrade/DistUpgradeController.py:385
msgid "Sandbox mode"
msgstr "Režim pieskoviska"

#: ../DistUpgrade/DistUpgradeController.py:386
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a system directory from now until the next reboot "
"are permanent."
msgstr ""
"Táto aktualizácia beží v režime pieskoviska (v testovacom režime). Všetky "
"zmeny zapísané do „%s“ sa pri ďalšom reštarte stratia.\n"
"\n"
"*Žiadne* zmeny zapísané do systémového adresára odteraz do najbližšieho "
"reštartu sa nezachovajú."

#: ../DistUpgrade/DistUpgradeController.py:414
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Vaša inštalácia Pythonu je pokazená. Prosím, opravte symbolický odkaz „/usr/"
"bin/python“."

#: ../DistUpgrade/DistUpgradeController.py:440
msgid "Package 'debsig-verify' is installed"
msgstr "Balík 'debsig-verify' je nainštalovaný"

#: ../DistUpgrade/DistUpgradeController.py:441
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Aktualizácia nemôže pokračovať s týmto nainštalovaným balíkom.\n"
"Najprv ho, prosím, odstránte pomocou programu synaptic alebo 'apt-get remove "
"debsig-verify' a spustite aktualizáciu znovu."

#: ../DistUpgrade/DistUpgradeController.py:453
#, python-format
msgid "Can not write to '%s'"
msgstr "Nie je možné zapisovať do  „%s“"

#: ../DistUpgrade/DistUpgradeController.py:454
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"Nie je možné zapisovať do systémového adresára  „%s“ na vašom systéme. "
"Aktualizácia nemôže pokračovať.\n"
"Prosím, uistite sa, že sa do systémového adresára dá zapisovať."

#: ../DistUpgrade/DistUpgradeController.py:465
msgid "Include latest updates from the Internet?"
msgstr "Použiť najnovšie aktualizácie z internetu?"

#: ../DistUpgrade/DistUpgradeController.py:466
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Pri aktualizácii systému je možné automaticky stiahnuť najnovšie "
"aktualizácie a nainštalovať ich. Ak ste pripojení k sieti, táto možnosť sa "
"dôrazne odporúča.\n"
"\n"
"Aktualizácia potrvá dlhšie, ale po jej dokončení bude váš systém úplne "
"aktuálny. Ak sa rozhodnete tento krok vynechať, mali by ste nainštalovať "
"najnovšie aktualizácie čo najskôr po aktualizácii systému.\n"
"Ak na túto možnosť odpoviete „Nie“, sieť nebude vôbec použitá."

#: ../DistUpgrade/DistUpgradeController.py:686
#, python-format
msgid "disabled on upgrade to %s"
msgstr "vypnuté pri aktualizácii na %s"

#: ../DistUpgrade/DistUpgradeController.py:713
msgid "No valid mirror found"
msgstr "Nebol nájdený vhodný server"

#: ../DistUpgrade/DistUpgradeController.py:714
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Počas prehľadávania vašich informácií o zdrojoch softvéru sa nenašlo žiadne "
"zrkadlo na aktualizáciu. To môže nastať ak používate lokálne zrkadlo alebo "
"ak sú informácie zrkadiel neaktuálne,\n"
"\n"
"Chcete napriek tomu prepísať svoj súbor „sources.list“? Ak tu zvolíte „Áno“, "
"prebehne aktualizácia všetkých „%s“ na záznamy „%s“.\n"
"Ak zvolíte „Nie“, aktualizácia sa preruší."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:734
msgid "Generate default sources?"
msgstr "Vytvoriť štandardný zoznam zdrojov softvéru?"

#: ../DistUpgrade/DistUpgradeController.py:735
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Po prehľadaní vášho súboru „sources.list“ sa nenašiel žiadny platný záznam "
"pre „%s“.\n"
"\n"
"Majú sa pridať štandardné záznamy pre „%s“? Ak zvolíte „Nie“, aktualizácia "
"sa preruší."

#: ../DistUpgrade/DistUpgradeController.py:770
msgid "Repository information invalid"
msgstr "Neplatná informácia o zdrojoch softvéru"

#: ../DistUpgrade/DistUpgradeController.py:771
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"Aktualizácia informácií zdroja softvéru spôsobila neplatný súbor, preto sa "
"spúšťa proces hlásenia chyby ."

#: ../DistUpgrade/DistUpgradeController.py:778
msgid "Third party sources disabled"
msgstr "Zdroje tretích strán sú zakázané"

#: ../DistUpgrade/DistUpgradeController.py:779
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Niektoré záznamy tretích strán vo vašom sources.list boli vypnuté. Po "
"prechode na novšiu verziu ich môžete znova zapnúť nástrojom 'software-"
"properties' vášho správcu balíkov."

#: ../DistUpgrade/DistUpgradeController.py:819
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Balíky sú v nekonzistentnom stave"
msgstr[1] "Balík je v nekonzistentnom stave"
msgstr[2] "Balíky sú v nekonzistentnom stave"

#: ../DistUpgrade/DistUpgradeController.py:822
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Balíky „%s“ sú v nekonzistentnom stave a je potrebné ich preinštalovať, ale "
"neboli pre ne nájdené žiadne archívy. Prosím, preinštalujte balíky ručne "
"alebo ich odstráňte zo systému."
msgstr[1] ""
"Balík „%s“ je v nekonzistentnom stave a je potrebné ich preinštalovať, ale "
"nemožno preň nájsť žiaden archív. Prosím, preinštalujte balík ručne alebo ho "
"odstráňte zo systému."
msgstr[2] ""
"Balíky „%s“ sú v nekonzistentnom stave a je potrebné ich preinštalovať, ale "
"neboli pre ne nájdené žiadne archívy. Prosím, preinštalujte balíky ručne "
"alebo ich odstráňte zo systému."

#: ../DistUpgrade/DistUpgradeController.py:870
msgid "Error during update"
msgstr "Chyba počas aktualizácie"

#: ../DistUpgrade/DistUpgradeController.py:871
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Počas aktualizácie sa objavil problém, ktorý je zvyčajne spôsobený chybou "
"sieťového pripojenia, skontrolujte prosím sieťové pripojenie a skúste to "
"znova."

#. print("on_button_install_clicked")
#: ../DistUpgrade/DistUpgradeController.py:880
#: ../UpdateManager/UpdateManager.py:757
msgid "Not enough free disk space"
msgstr "Nedostatok voľného miesta na disku"

#: ../DistUpgrade/DistUpgradeController.py:881
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"Aktualizácia bola prerušená. Aktualizácia vyžaduje celkom %s voľného miesta "
"na disku „%s“. Prosím, uvoľnite ešte aspoň %s miesta na disku „%s“. "
"Vyprázdnite svoj kôš a odstráňte dočasné balíky z predošlých inštalácií "
"pomocou „sudo apt-get clean“."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:910
#: ../DistUpgrade/DistUpgradeController.py:1692
msgid "Calculating the changes"
msgstr "Počítajú sa zmeny"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:942
msgid "Do you want to start the upgrade?"
msgstr "Chcete začať s aktualizáciou?"

#: ../DistUpgrade/DistUpgradeController.py:1008
msgid "Upgrade canceled"
msgstr "Aktualizácia zrušená"

#: ../DistUpgrade/DistUpgradeController.py:1009
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Aktualizácia sa teraz preruší a obnoví sa pôvodný stav systému. V "
"aktualizácii môžete neskôr pokračovať."

#: ../DistUpgrade/DistUpgradeController.py:1015
#: ../DistUpgrade/DistUpgradeController.py:1149
msgid "Could not download the upgrades"
msgstr "Nebolo možné stiahnuť aktualizácie"

#: ../DistUpgrade/DistUpgradeController.py:1016
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Aktualizácia bola zrušená. Prosím, skontrolujte funkčnosť vášho pripojenia k "
"internetu alebo inštalačné médium a skúste to znova. Všetky stiahnuté súbory "
"boli zachované."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1100
#: ../DistUpgrade/DistUpgradeController.py:1137
#: ../DistUpgrade/DistUpgradeController.py:1242
msgid "Error during commit"
msgstr "Chyba počas potvrdzovania"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1102
#: ../DistUpgrade/DistUpgradeController.py:1139
#: ../DistUpgrade/DistUpgradeController.py:1281
msgid "Restoring original system state"
msgstr "Obnovuje sa pôvodný stav systému"

#: ../DistUpgrade/DistUpgradeController.py:1103
#: ../DistUpgrade/DistUpgradeController.py:1118
#: ../DistUpgrade/DistUpgradeController.py:1140
msgid "Could not install the upgrades"
msgstr "Nebolo možné nainštalovať aktualizácie"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1108
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Aktualizácia bola prerušená. Váš systém sa môže nachádzať v nepoužiteľnom "
"stave. Teraz sa spustí pokus o obnovenie (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1113
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/update-manager/+filebug and attach the files in /var/log/dist-"
"upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Prosím, použite prehliadač na ohlásenie tejto chyby na adrese http://bugs."
"launchpad.net/ubuntu/+source/update-manager/+filebug a pripojte k chybovému "
"hláseniu súbory v /var/log/dist-upgrade/.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1150
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Aktualizácia bola prerušená. Prosím, skontrolujte svoje pripojenie k "
"internetu alebo inštalačné médium a skúste to znova. "

#: ../DistUpgrade/DistUpgradeController.py:1230
msgid "Remove obsolete packages?"
msgstr "Odstrániť zastarané balíky?"

#: ../DistUpgrade/DistUpgradeController.py:1231
#: ../DistUpgrade/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Ponechať"

#: ../DistUpgrade/DistUpgradeController.py:1231
msgid "_Remove"
msgstr "_Odstrániť"

#: ../DistUpgrade/DistUpgradeController.py:1243
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Počas čistenia nastal problém. Viac informácií nájdete v správe uvedenej "
"nižšie. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1319
msgid "Required depends is not installed"
msgstr "Požadované závislosti nie sú nainštalované"

#: ../DistUpgrade/DistUpgradeController.py:1320
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Požadovaná závislosť '%s' nie je nainštalovaná. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1588
#: ../DistUpgrade/DistUpgradeController.py:1653
msgid "Checking package manager"
msgstr "Kontroluje sa správca balíkov"

#: ../DistUpgrade/DistUpgradeController.py:1593
msgid "Preparing the upgrade failed"
msgstr "Príprava prechodu na vyššiu verziu zlyhala"

#: ../DistUpgrade/DistUpgradeController.py:1594
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""
"Príprava systému na aktualizáciu zlyhala,  preto sa spúšťa proces hlásenia "
"chyby ."

#: ../DistUpgrade/DistUpgradeController.py:1608
msgid "Getting upgrade prerequisites failed"
msgstr "Zisťovanie predpokladov aktualizácie zlyhalo"

#: ../DistUpgrade/DistUpgradeController.py:1609
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"Systému sa nepodarilo získať predpoklady  aktualizácie. Aktualizácia sa "
"teraz preruší a obnoví sa pôvodný stav systému.\n"
"\n"
"Okrem toho sa spúšťa proces hlásenia chyby ."

#: ../DistUpgrade/DistUpgradeController.py:1637
msgid "Updating repository information"
msgstr "Aktualizujú sa informácie o zdrojoch softvéru"

#: ../DistUpgrade/DistUpgradeController.py:1644
msgid "Failed to add the cdrom"
msgstr "Nepodarilo sa pridať CD-ROM"

#: ../DistUpgrade/DistUpgradeController.py:1645
msgid "Sorry, adding the cdrom was not successful."
msgstr "Ľutujeme, pridanie CD-ROM neprebehlo úspešne"

#: ../DistUpgrade/DistUpgradeController.py:1673
msgid "Invalid package information"
msgstr "Neplatná informácia o balíku"

#: ../DistUpgrade/DistUpgradeController.py:1674
msgid "After updating your package "
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1698
#: ../DistUpgrade/DistUpgradeController.py:1750
msgid "Fetching"
msgstr "Sťahuje sa"

#: ../DistUpgrade/DistUpgradeController.py:1704
#: ../DistUpgrade/DistUpgradeController.py:1754
msgid "Upgrading"
msgstr "Prebieha prechod na novšiu verziu"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1709
#: ../DistUpgrade/DistUpgradeController.py:1756
#: ../DistUpgrade/DistUpgradeController.py:1763
#: ../DistUpgrade/DistUpgradeController.py:1774
msgid "Upgrade complete"
msgstr "Aktualizácia dokončená"

#: ../DistUpgrade/DistUpgradeController.py:1710
#: ../DistUpgrade/DistUpgradeController.py:1757
#: ../DistUpgrade/DistUpgradeController.py:1764
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "Aktualizácia bola dokončená, ale počas nej sa vyskytli chyby."

#: ../DistUpgrade/DistUpgradeController.py:1717
msgid "Searching for obsolete software"
msgstr "Vyhľadávanie zastaraného softvéru"

#: ../DistUpgrade/DistUpgradeController.py:1726
msgid "System upgrade is complete."
msgstr "Aktualizácia systému je dokončená."

#: ../DistUpgrade/DistUpgradeController.py:1775
msgid "The partial upgrade was completed."
msgstr "Čiastočná aktualizácia je dokončená."

#: ../DistUpgrade/DistUpgradeQuirks.py:204
msgid "evms in use"
msgstr "používa sa evms"

#: ../DistUpgrade/DistUpgradeQuirks.py:205
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"Váš systém používa správcu zväzkov „evms“ v /proc/mounts. Program „evms“ už "
"ďalej nie je podporovaný. Vypnite ho prosím a následne znova spustite "
"aktualizáciu."

#: ../DistUpgrade/DistUpgradeQuirks.py:502
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr "Ubuntu 12.04 LTS nemusí plne podporovať váš grafický hardvér."

#: ../DistUpgrade/DistUpgradeQuirks.py:504
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"Podpora vášho grafického hardvéru Intel je v Ubuntu 12.04 LTS obmedzená a po "
"aktualizácii sa môžu vyskytnúť problémy. Ďalšie informácie nájdete na "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Chcete pokračovať "
"v aktualizácii?"

#: ../DistUpgrade/DistUpgradeQuirks.py:526
#: ../DistUpgrade/DistUpgradeQuirks.py:554
#: ../DistUpgrade/DistUpgradeQuirks.py:581
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Aktualizácia systému môže vypnúť niektoré efekty prostredia a znížiť výkon "
"hier a iných graficky náročných programov."

#: ../DistUpgrade/DistUpgradeQuirks.py:530
#: ../DistUpgrade/DistUpgradeQuirks.py:558
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Tento počítač momentálne používa grafický ovládač NVIDIA „nvidia“. Nie je "
"dostupná žiadna verzia tohto ovládača, ktorá by fungovala s vaším hardvérom "
"v Ubuntu 10.04 LTS.\n"
"\n"
"Chcete pokračovať?"

#: ../DistUpgrade/DistUpgradeQuirks.py:585
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Tento počítač momentálne používa grafický ovládač AMD „fglrx“. Nie je "
"dostupná žiadna verzia tohto ovládača, ktorá by fungovala s vaším hardvérom "
"v Ubuntu 10.04 LTS.\n"
"\n"
"Chcete pokračovať?"

#: ../DistUpgrade/DistUpgradeQuirks.py:615
msgid "No i686 CPU"
msgstr "CPU nie je i686"

#: ../DistUpgrade/DistUpgradeQuirks.py:616
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Váš systém má CPU i586 alebo CPU bez rozšírenia „cmov“. Všetky balíky boli "
"vytvorené a optimalizované pre procesory i686 alebo vyššie. Nie je možné "
"nainštalovať nové vydanie Ubuntu na tento hardvér."

#: ../DistUpgrade/DistUpgradeQuirks.py:652
msgid "No ARMv6 CPU"
msgstr "Neobsahuje procesor ARMv6"

#: ../DistUpgrade/DistUpgradeQuirks.py:653
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Váš systém používa procesor ARM, ktorý je staršej architektúry ako ARMv6. "
"Všetky balíky v karmic boli zostavené s optimalizáciami vyžadujúcimi "
"architektúru ARMv6 alebo lepšiu. Na tomto hardvéri nie je možné aktualizovať "
"váš systém na nové vydanie Ubuntu."

#: ../DistUpgrade/DistUpgradeQuirks.py:673
msgid "No init available"
msgstr "Nie je dostupný žiaden init"

#: ../DistUpgrade/DistUpgradeQuirks.py:674
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Zdá sa, že váš systém  je virtualizované prostredie bez démona init, napr. "
"Linux-VServer. Ubuntu 10.04 LTS nemôže v tomto type prostredia fungovať, "
"najskôr je potrebná aktualizácia vášho virtuálneho stroja.\n"
"\n"
"Ste si istý, že chcete pokračovať?"

#: ../DistUpgrade/DistUpgradeMain.py:65
msgid "Sandbox upgrade using aufs"
msgstr "Aktualizácia v pieskovisku pomocou aufs"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "Použiť zadanú cestu na hľadanie CD s aktualizačnými balíkami"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Použiť frontend. Momentálne dostupné: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:76
msgid "*DEPRECATED* this option will be ignored"
msgstr "*ZAVRHOVANÉ* táto voľba bude ignorovaná"

#: ../DistUpgrade/DistUpgradeMain.py:79
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "Vykonať iba čiastočnú aktualizáciu (bez prepísania sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:82
msgid "Disable GNU screen support"
msgstr "Vypnúť podporu GNU screen"

#: ../DistUpgrade/DistUpgradeMain.py:84
msgid "Set datadir"
msgstr "Nastaviť dátový priečinok"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeViewGtk.py:114
#: ../DistUpgrade/DistUpgradeViewGtk3.py:117
#: ../DistUpgrade/DistUpgradeViewKDE.py:195
#: ../UpdateManager/DistUpgradeFetcherKDE.py:155
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Prosím, vložte „%s“ do mechaniky „%s“"

#: ../DistUpgrade/DistUpgradeViewGtk.py:135
#: ../DistUpgrade/DistUpgradeViewGtk3.py:138
#: ../DistUpgrade/DistUpgradeViewKDE.py:209
msgid "Fetching is complete"
msgstr "Sťahovanie je dokončené"

#: ../DistUpgrade/DistUpgradeViewGtk.py:146
#: ../DistUpgrade/DistUpgradeViewGtk3.py:149
#: ../DistUpgrade/DistUpgradeViewKDE.py:222
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Získava sa súbor %li z %li rýchlosťou %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:149
#: ../DistUpgrade/DistUpgradeViewGtk.py:296
#: ../DistUpgrade/DistUpgradeViewGtk3.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:309
#: ../DistUpgrade/DistUpgradeViewKDE.py:223
#: ../DistUpgrade/DistUpgradeViewKDE.py:371
#, python-format
msgid "About %s remaining"
msgstr "Zostáva približne %s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:155
#: ../DistUpgrade/DistUpgradeViewKDE.py:225
#, python-format
msgid "Fetching file %li of %li"
msgstr "Sťahuje sa súbor %li z %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:183
#: ../DistUpgrade/DistUpgradeViewGtk3.py:186
#: ../DistUpgrade/DistUpgradeViewKDE.py:262
msgid "Applying changes"
msgstr "Aplikujú sa zmeny"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:208
#: ../DistUpgrade/DistUpgradeViewGtk3.py:212
#: ../DistUpgrade/DistUpgradeViewKDE.py:275
msgid "dependency problems - leaving unconfigured"
msgstr "problém so závislosťami - ponecháva sa nenakonfigurované"

#: ../DistUpgrade/DistUpgradeViewGtk.py:213
#: ../DistUpgrade/DistUpgradeViewGtk3.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:277
#, python-format
msgid "Could not install '%s'"
msgstr "Nebolo možné nainštalovať „%s“"

#: ../DistUpgrade/DistUpgradeViewGtk.py:214
#: ../DistUpgrade/DistUpgradeViewGtk3.py:218
#: ../DistUpgrade/DistUpgradeViewKDE.py:278
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Aktualizácia bude pokračovať,  ale balík „%s“ nemusí byť vo fungujúcom "
"stave. Zvážte prosím zaslanie hlásenia o chybe."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:231
#: ../DistUpgrade/DistUpgradeViewGtk3.py:235
#: ../DistUpgrade/DistUpgradeViewKDE.py:299
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Nahradiť upravený konfiguračný súbor\n"
"„%s“?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:232
#: ../DistUpgrade/DistUpgradeViewGtk3.py:236
#: ../DistUpgrade/DistUpgradeViewKDE.py:300
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Ak si vyberiete nahradiť novšou verziou, stratíte všetky zmeny, ktoré ste "
"spravili v tejto konfigurácii."

#: ../DistUpgrade/DistUpgradeViewGtk.py:251
#: ../DistUpgrade/DistUpgradeViewGtk3.py:256
#: ../DistUpgrade/DistUpgradeViewKDE.py:323
msgid "The 'diff' command was not found"
msgstr "Príkaz „diff“ nebol nájdený."

#: ../DistUpgrade/DistUpgradeViewGtk.py:464
#: ../DistUpgrade/DistUpgradeViewGtk3.py:477
#: ../DistUpgrade/DistUpgradeViewText.py:92
msgid "A fatal error occurred"
msgstr "Vyskytla sa nenapraviteľná chyba"

#: ../DistUpgrade/DistUpgradeViewGtk.py:465
#: ../DistUpgrade/DistUpgradeViewGtk3.py:478
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Prosím, nahláste toto ako chybu (ak ste tak už neučinili) a priložte súbory /"
"var/log/dist-upgrade/main.log a /var/log/dist-upgrade/apt.log k chybovému "
"hláseniu. Aktualizácia bola prerušená.\n"
"Váš pôvodný súbor sources.list bol uložený ako /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk.py:482
#: ../DistUpgrade/DistUpgradeViewGtk3.py:495
msgid "Ctrl-c pressed"
msgstr "Stlačené Ctrl-c"

#: ../DistUpgrade/DistUpgradeViewGtk.py:483
#: ../DistUpgrade/DistUpgradeViewGtk3.py:496
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Týmto sa operácia preruší a môže ponechať systém v nepoužiteľnom stave. Ste "
"si istý, že to chcete?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:631
#: ../DistUpgrade/DistUpgradeViewGtk3.py:629
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Aby ste zamedzili strate údajov, zatvorte všetky otvorené aplikácie a "
"dokumenty."

#: ../DistUpgrade/DistUpgradeViewGtk.py:645
#: ../DistUpgrade/DistUpgradeViewGtk3.py:643
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Tento balík už Canonical nepodporuje (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:646
#: ../DistUpgrade/DistUpgradeViewGtk3.py:644
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Znížiť verziu (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:647
#: ../DistUpgrade/DistUpgradeViewGtk3.py:645
#, python-format
msgid "Remove (%s)"
msgstr "Odstrániť (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:648
#: ../DistUpgrade/DistUpgradeViewGtk3.py:646
#, python-format
msgid "No longer needed (%s)"
msgstr "Už nie je potrebné (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:649
#: ../DistUpgrade/DistUpgradeViewGtk3.py:647
#, python-format
msgid "Install (%s)"
msgstr "Nainštalovať (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:650
#: ../DistUpgrade/DistUpgradeViewGtk3.py:648
#, python-format
msgid "Upgrade (%s)"
msgstr "Aktualizovať (%s)"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:196
#: ../UpdateManager/DistUpgradeFetcherKDE.py:157
msgid "Media Change"
msgstr "Výmena nosiča"

#: ../DistUpgrade/DistUpgradeViewKDE.py:335
msgid "Show Difference >>>"
msgstr "Zobraziť rozdiel >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:338
msgid "<<< Hide Difference"
msgstr "<<< Skryť rozdiel"

#: ../DistUpgrade/DistUpgradeViewKDE.py:554
msgid "Error"
msgstr "Chyba"

#: ../DistUpgrade/DistUpgradeViewKDE.py:568
msgid "&Cancel"
msgstr "&Zrušiť"

#: ../DistUpgrade/DistUpgradeViewKDE.py:572
#: ../DistUpgrade/DistUpgradeViewKDE.py:813
msgid "&Close"
msgstr "&Zavrieť"

#: ../DistUpgrade/DistUpgradeViewKDE.py:618
msgid "Show Terminal >>>"
msgstr "Zobraziť Terminál >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:621
msgid "<<< Hide Terminal"
msgstr "<<< Skryť Terminál"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Information"
msgstr "Informácie"

#: ../DistUpgrade/DistUpgradeViewKDE.py:751
#: ../DistUpgrade/DistUpgradeViewKDE.py:796
#: ../DistUpgrade/DistUpgradeViewKDE.py:799 ../DistUpgrade/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Podrobnosti"

#: ../DistUpgrade/DistUpgradeViewKDE.py:777
#, python-format
msgid "No longer supported %s"
msgstr "Už nie je podporované %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:779
#, python-format
msgid "Remove %s"
msgstr "Odstrániť %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:781
#: ../DistUpgrade/DistUpgradeViewText.py:182
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Odstrániť (bol nainštalovaný automaticky) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:783
#, python-format
msgid "Install %s"
msgstr "Inštalovať %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:785
#, python-format
msgid "Upgrade %s"
msgstr "Aktualizovať %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:809
#: ../DistUpgrade/DistUpgradeViewText.py:230
msgid "Restart required"
msgstr "Je potrebný reštart"

#: ../DistUpgrade/DistUpgradeViewKDE.py:809
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Pre dokončenie aktualizácie reštartujte počítač</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:812 ../DistUpgrade/DistUpgrade.ui.h:14
#: ../data/gtkbuilder/UpdateManager.ui.h:26
msgid "_Restart Now"
msgstr "_Reštartovať teraz"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:830
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Zrušiť prebiehajúcu aktualizáciu systému?</big></b>\n"
"\n"
"Ak zrušíte prebiehajúcu aktualizáciu, môže to ponechať systém v "
"nepoužiteľnom stave. Dôrazne sa odporúča pokračovať v aktualizácii systému."

#: ../DistUpgrade/DistUpgradeViewKDE.py:834
msgid "Cancel Upgrade?"
msgstr "Zrušiť prechod na novšiu verziu?"

#: ../DistUpgrade/DistUpgradeView.py:61
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li dní"
msgstr[1] "deň"
msgstr[2] "%li dni"

#: ../DistUpgrade/DistUpgradeView.py:63
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li hodín"
msgstr[1] "hodina"
msgstr[2] "%li hodiny"

#: ../DistUpgrade/DistUpgradeView.py:65
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minút"
msgstr[1] "minúta"
msgstr[2] "%li minúty"

#: ../DistUpgrade/DistUpgradeView.py:66
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li sekúnd"
msgstr[1] "sekunda"
msgstr[2] "%li sekundy"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s, %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:100
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:151
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Sťahovanie potrvá asi %s na 1Mbit DSL pripojení a asi %s na 56k modeme."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:155
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Sťahovanie s vaším pripojením bude trvať asi %s. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:259 ../DistUpgrade/DistUpgrade.ui.h:21
msgid "Preparing to upgrade"
msgstr "Prebieha príprava aktualizácie"

#: ../DistUpgrade/DistUpgradeView.py:260
msgid "Getting new software channels"
msgstr "Získavajú sa softvérové kanály"

#: ../DistUpgrade/DistUpgradeView.py:261 ../DistUpgrade/DistUpgrade.ui.h:23
msgid "Getting new packages"
msgstr "Prijímajú sa nové balíky"

#: ../DistUpgrade/DistUpgradeView.py:262 ../DistUpgrade/DistUpgrade.ui.h:26
msgid "Installing the upgrades"
msgstr "Inštalujú sa aktualizácie"

#: ../DistUpgrade/DistUpgradeView.py:263 ../DistUpgrade/DistUpgrade.ui.h:25
msgid "Cleaning up"
msgstr "Prebieha čistenie"

#: ../DistUpgrade/DistUpgradeView.py:348
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d nainštalovaných balíkov už Canonical nepodporuje. Ešte stále "
"môžete získať podporu od komunity."
msgstr[1] ""
"%(amount)d nainštalovaný balík už Canonical nepodporuje. Ešte stále môžete "
"získať podporu od komunity."
msgstr[2] ""
"%(amount)d nainštalované balíky už Canonical nepodporuje. Ešte stále môžete "
"získať podporu od komunity."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:357
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "Bude odstránených %d balíkov."
msgstr[1] "Bude odstránený %d balík."
msgstr[2] "Budú odstránené %d balíky."

#: ../DistUpgrade/DistUpgradeView.py:362
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "Bude nainštalovaných %d nových balíkov."
msgstr[1] "Bude nainštalovaný %d nový balík."
msgstr[2] "Budú nainštalované %d nové balíky."

#: ../DistUpgrade/DistUpgradeView.py:368
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "Bude aktualizovaných %d balíkov."
msgstr[1] "Bude aktualizovaný %d balík."
msgstr[2] "Budú aktualizované %d balíky."

#: ../DistUpgrade/DistUpgradeView.py:373
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Musíte stiahnuť celkom %s. "

#: ../DistUpgrade/DistUpgradeView.py:378
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"Inštalácia aktualizácie môže trvať niekoľko hodín. Po dokončení sťahovania "
"nebude možné proces zrušiť."

#: ../DistUpgrade/DistUpgradeView.py:382
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"Sťahovanie a inštalácia aktualizácií môže trvať niekoľko hodín. Po skončení "
"sťahovania nie je možné proces zrušiť."

#: ../DistUpgrade/DistUpgradeView.py:387
msgid "Removing the packages can take several hours. "
msgstr "odstraňovanie balíkov môže trvať niekoľko hodín. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:392 ../UpdateManager/UpdateManager.py:676
msgid "The software on this computer is up to date."
msgstr "Softvér na tomto počítači je aktuálny."

#: ../DistUpgrade/DistUpgradeView.py:393
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Pre váš systém nie sú dostupné žiadne aktualizácie. Proces aktualizácie bude "
"zrušený."

#: ../DistUpgrade/DistUpgradeView.py:406
msgid "Reboot required"
msgstr "Je potrebný reštart"

#: ../DistUpgrade/DistUpgradeView.py:407
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Bola dokončená aktualizácia a je potrebné reštartovať počítač. Chcete "
"vykonať reštart teraz?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:72
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:72
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "overenie „%(file)s“ voči „%(signature)s“ "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:131
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:131
#, python-format
msgid "extracting '%s'"
msgstr "extrahuje sa „%s“"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:151
#: ../DistUpgrade/DistUpgradeFetcherCore.py:152
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:151
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:152
msgid "Could not run the upgrade tool"
msgstr "Nebolo možné spustiť aktualizačný program"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:152
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:152
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug update-manager'."
msgstr ""
"Toto je pravdepodobne chyba v nástroji na aktualizáciu. Prosím, nahláste to "
"ako chybu príkazom „ubuntu-bug update-manager“."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:227
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:227
msgid "Upgrade tool signature"
msgstr "Podpis aktualizačného programu"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:234
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:234
msgid "Upgrade tool"
msgstr "Aktualizačný nástroj"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:268
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:268
msgid "Failed to fetch"
msgstr "Zlyhalo získavanie"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:269
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:269
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Zlyhalo získavanie aktualizácie. Môže to byť spôsobené sieťovým problémom. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:273
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:273
msgid "Authentication failed"
msgstr "Overenie totožnosti zlyhalo"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:274
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:274
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Zlyhalo overenie pravosti aktualizácie. Môže to byť spôsobené sieťovým "
"problémom alebo nedostupnosťou servera. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:279
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:279
msgid "Failed to extract"
msgstr "Chyba pri rozbaľovaní"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:280
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:280
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Nebolo možné rozbaliť aktualizáciu. Môže to byť spôsobené sieťovým problémom "
"alebo nedostupnosťou servera. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:285
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:285
msgid "Verification failed"
msgstr "Overenie zlyhalo"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:286
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:286
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Zlyhalo overenie aktualizácie. Mohol to spôsobiť problém siete alebo "
"servera. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:300
#: ../DistUpgrade/DistUpgradeFetcherCore.py:306
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:300
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:306
msgid "Can not run the upgrade"
msgstr "Nie je možné spustiť aktualizáciu systému"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:301
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:301
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"To zvyčajne spôsobuje systém, kde je /tmp pripojený s príznakom noexec. "
"Prosím, znova ho pripojte bez príznaku noexec a znova spustite aktualizáciu."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:307
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:307
#, python-format
msgid "The error message is '%s'."
msgstr "Chybová správa je '%s'."

#: ../DistUpgrade/DistUpgradeViewText.py:93
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Prosím, nahláste toto ako chybu a priložte súbory /var/log/dist-upgrade/main."
"log a /var/log/dist-upgrade/apt.log k chybovému hláseniu. Aktualizácia bola "
"prerušená.\n"
"Váš pôvodný súbor sources.list bol uložený ako /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:117
msgid "Aborting"
msgstr "Ruší sa"

#: ../DistUpgrade/DistUpgradeViewText.py:122
msgid "Demoted:\n"
msgstr "Znížená verzia:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:129
msgid "To continue please press [ENTER]"
msgstr "Prosím, pokračujte stlačením [ENTER]"

#: ../DistUpgrade/DistUpgradeViewText.py:157
#: ../DistUpgrade/DistUpgradeViewText.py:196
#: ../DistUpgrade/DistUpgradeViewText.py:203
msgid "Continue [yN] "
msgstr "Pokračovať [aN] "

#: ../DistUpgrade/DistUpgradeViewText.py:157
#: ../DistUpgrade/DistUpgradeViewText.py:196
msgid "Details [d]"
msgstr "Podrobnosti [p]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:162
#: ../DistUpgrade/DistUpgradeViewText.py:206
msgid "y"
msgstr "a"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:165
#: ../DistUpgrade/DistUpgradeViewText.py:213
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:168
msgid "d"
msgstr "p"

#: ../DistUpgrade/DistUpgradeViewText.py:173
#, python-format
msgid "No longer supported: %s\n"
msgstr "Už nie je podporované: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
#, python-format
msgid "Remove: %s\n"
msgstr "Odstrániť: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:188
#, python-format
msgid "Install: %s\n"
msgstr "Inštalovať: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:193
#, python-format
msgid "Upgrade: %s\n"
msgstr "Aktualizovať: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:210
msgid "Continue [Yn] "
msgstr "Pokračovať [An] "

#: ../DistUpgrade/DistUpgradeViewText.py:231
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Pre dokončenie aktualizácie je vyžadovaný reštart.\n"
"Ak zvolíte 'y' systém sa reštartuje."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Zrušiť aktualizáciu"

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "_Pokračovať v aktualizácii"

#: ../DistUpgrade/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Zrušiť prebiehajúcu aktualizáciu?</big></b>\n"
"\n"
"Ak prerušíte aktualizáciu, systém môže zostať v nestabilnom stave. Dôrazne "
"sa odporúča pokračovať v aktualizácii."

#: ../DistUpgrade/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Začať aktualizáciu"

#: ../DistUpgrade/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Nahradiť"

#: ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Rozdiel medzi súbormi"

#: ../DistUpgrade/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Oznámiť chybu"

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Pokračovať"

#: ../DistUpgrade/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Spustiť aktualizáciu?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Reštart systému aby sa dokončila aktualizácia</big></b>\n"
"\n"
"Prosím, uložte rozrobenú prácu predtým, než budete pokračovať."

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Prechod na vyššiu verziu distribúcie"

#: ../DistUpgrade/DistUpgrade.ui.h:19
#, fuzzy
msgid "<b><big>Upgrading Ubuntu to version 12.10</big></b>"
msgstr "<b><big>Aktualizácia Ubuntu na verziu 11.10</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "Setting new software channels"
msgstr "Nastavujú sa softvérové kanály"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "Restarting the computer"
msgstr "Reštartovanie systému"

#: ../DistUpgrade/DistUpgrade.ui.h:27
msgid "Terminal"
msgstr "Terminál"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:64
msgid "Please wait, this can take some time."
msgstr "Čakajte prosím, toto môže chvíľu trvať."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:66
msgid "Update is complete"
msgstr "Aktualizácia je dokončená"

#: ../UpdateManager/DistUpgradeFetcher.py:114
#: ../UpdateManager/DistUpgradeFetcherKDE.py:109
msgid "Could not find the release notes"
msgstr "Nebolo možné nájsť poznámky k vydaniu"

#: ../UpdateManager/DistUpgradeFetcher.py:115
#: ../UpdateManager/DistUpgradeFetcherKDE.py:110
msgid "The server may be overloaded. "
msgstr "Server môže byť preťažený. "

#: ../UpdateManager/DistUpgradeFetcher.py:125
#: ../UpdateManager/DistUpgradeFetcherKDE.py:114
msgid "Could not download the release notes"
msgstr "Nebolo možné stiahnuť poznámky k vydaniu"

#: ../UpdateManager/DistUpgradeFetcher.py:126
#: ../UpdateManager/DistUpgradeFetcherKDE.py:115
msgid "Please check your internet connection."
msgstr "Skontrolujte si internetové pripojenie."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:68
#: ../UpdateManager/DistUpgradeFetcherKDE.py:91
msgid "Upgrade"
msgstr "Aktualizovať"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:95
#: ../data/gtkbuilder/UpdateManager.ui.h:20
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Release Notes"
msgstr "Poznámky k vydaniu"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:134
#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
msgid "Downloading additional package files..."
msgstr "Sťahujú sa ďalšie balíky..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Súbor %s z %s, %sB/s"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
#, python-format
msgid "File %s of %s"
msgstr "Súbor %s z %s"

#: ../UpdateManager/ChangelogViewer.py:75
msgid "Open Link in Browser"
msgstr "Otvoriť odkaz v prehliadači"

#: ../UpdateManager/ChangelogViewer.py:78
msgid "Copy Link to Clipboard"
msgstr "Skopírovať odkaz do schránky"

#: ../UpdateManager/GtkProgress.py:162
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Sťahuje sa súbor %(current)li z %(total)li s %(speed)s/s"

#: ../UpdateManager/GtkProgress.py:167
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Sťahuje sa súbor %(current)li z %(total)li"

#: ../UpdateManager/UpdateManager.py:106 ../do-release-upgrade:100
msgid "Your Ubuntu release is not supported anymore."
msgstr "Vaše vydanie Ubuntu už nie je podporované."

#: ../UpdateManager/UpdateManager.py:107
msgid ""
"You will not get any further security fixes or critical updates. Please "
"upgrade to a later version of Ubuntu."
msgstr ""
"Nebudete dostávať žiadne ďalšie bezpečnostné opravy ani dôležité "
"aktualizácie. Prosím, aktualizujte na novšiu verziu Ubuntu."

#: ../UpdateManager/UpdateManager.py:115
msgid "Upgrade information"
msgstr "Informácie o prechode na novšiu verziu"

#: ../UpdateManager/UpdateManager.py:233
#: ../UpdateManagerText/UpdateManagerText.py:35
msgid "Install"
msgstr "Nainštalovať"

#: ../UpdateManager/UpdateManager.py:235
msgid "Name"
msgstr "Názov"

#. upload_archive = version_match.group(2).strip()
#: ../UpdateManager/UpdateManager.py:395
#, python-format
msgid "Version %s: \n"
msgstr "Verzia %s: \n"

#: ../UpdateManager/UpdateManager.py:453
msgid ""
"No network connection detected, you can not download changelog information."
msgstr ""
"Nebolo nájdené žiadne pripojenie k sieti, preto nie je možné stiahnuť záznam "
"zmien."

#: ../UpdateManager/UpdateManager.py:463
msgid "Downloading list of changes..."
msgstr "Sťahuje sa zoznam zmien..."

#: ../UpdateManager/UpdateManager.py:507
msgid "_Deselect All"
msgstr "_Zrušiť výber všetkých"

#: ../UpdateManager/UpdateManager.py:513
msgid "Select _All"
msgstr "Vybrať _všetky"

#: ../UpdateManager/UpdateManager.py:572
#, python-format
msgid "%s will be downloaded."
msgstr "%s sa stiahne."

#: ../UpdateManager/UpdateManager.py:584
#, fuzzy
msgid "The update has already been downloaded."
msgid_plural "The updates have already been downloaded."
msgstr[0] "Aktualizácie už boli stiahnuté, ale nie nainštalované."
msgstr[1] "Aktualizácia už boli stiahnutá, ale nie nainštalovaná."
msgstr[2] "Aktualizácie už boli stiahnuté, ale nie nainštalované."

#: ../UpdateManager/UpdateManager.py:589
msgid "There are no updates to install."
msgstr "Nie sú dostupné žiadne aktualizácie na inštaláciu."

#: ../UpdateManager/UpdateManager.py:598
msgid "Unknown download size."
msgstr "Neznáma veľkosť na stiahnutie."

#: ../UpdateManager/UpdateManager.py:624
msgid ""
"It is unknown when the package information was updated last. Please click "
"the 'Check' button to update the information."
msgstr ""
"Nie je známe, kedy boli informácie balíka naposledy aktualizované. Prosím, "
"aktualizujte informácie kliknutím  na tlačidlo „Skontrolovať“."

#: ../UpdateManager/UpdateManager.py:630
#, python-format
msgid ""
"The package information was last updated %(days_ago)s days ago.\n"
"Press the 'Check' button below to check for new software updates."
msgstr ""
"Informácie o balíkoch boli naposledy aktualizované pred %(days_ago)s dňami.\n"
"Nové aktualizácie môžete skontrolovať teraz stlačením tlačidla "
"„Skontrolovať“."

#: ../UpdateManager/UpdateManager.py:635
#, python-format
msgid "The package information was last updated %(days_ago)s day ago."
msgid_plural "The package information was last updated %(days_ago)s days ago."
msgstr[0] ""
"Informácie o balíkoch boli naposledy aktualizované pred %(days_ago)s dňami."
msgstr[1] ""
"Informácie o balíkoch boli naposledy aktualizované pred %(days_ago)s dňom."
msgstr[2] ""
"Informácie o balíkoch boli naposledy aktualizované pred %(days_ago)s dňami."

#: ../UpdateManager/UpdateManager.py:639
#, python-format
msgid "The package information was last updated %(hours_ago)s hour ago."
msgid_plural ""
"The package information was last updated %(hours_ago)s hours ago."
msgstr[0] ""
"Informácie o balíkoch boli naposledy aktualizované pred %(hours_ago)s "
"hodinami."
msgstr[1] ""
"Informácie o balíkoch boli naposledy aktualizované pred %(hours_ago)s "
"hodinou."
msgstr[2] ""
"Informácie o balíkoch boli naposledy aktualizované pred %(hours_ago)s "
"hodinami."

#. TRANSLATORS: only in plural form, as %s minutes ago is one of 15, 30, 45 minutes ago
#: ../UpdateManager/UpdateManager.py:644 ../UpdateManager/UpdateManager.py:646
#: ../UpdateManager/UpdateManager.py:648
#, python-format
msgid "The package information was last updated about %s minutes ago."
msgstr "Informácie balíka naposledy aktualizované pred %s minútami."

#: ../UpdateManager/UpdateManager.py:650
msgid "The package information was just updated."
msgstr "Informácie balíka boli práve aktualizované."

#: ../UpdateManager/UpdateManager.py:689
msgid "Software updates may be available for your computer."
msgstr "Pre váš počítač môžu byť dostupné aktualizácie softvéru."

#: ../UpdateManager/UpdateManager.py:697
#, python-format
msgid ""
"Updated software has been issued since %s was released. Do you want to "
"install it now?"
msgstr ""

#: ../UpdateManager/UpdateManager.py:700
msgid ""
"Updated software is available for this computer. Do you want to install it "
"now?"
msgstr ""

#: ../UpdateManager/UpdateManager.py:758
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"Aktualizácia vyžaduje %s voľného miesta na disku „%s“. Prosím, uvoľnite "
"aspoň ďalších %s miesta na disku „%s“. Vyprázdnite odpadkový kôš a odstráňte "
"dočasné balíky z predošlých aktualizácií príkazom „sudo apt-get clean“."

#: ../UpdateManager/UpdateManager.py:783
msgid ""
"The computer needs to restart to finish installing updates. Please save your "
"work before continuing."
msgstr ""
"Je potrebné reštartovať počítač aby sa dokončila inštalácia aktualizácií. "
"Prosím, uložte svoju prácu než budete pokračovať."

#: ../UpdateManager/UpdateManager.py:847
msgid "Reading package information"
msgstr "Načítavajú sa informácie o balíku"

#: ../UpdateManager/UpdateManager.py:862
msgid "Connecting..."
msgstr "Pripája sa..."

#: ../UpdateManager/UpdateManager.py:879
msgid "You may not be able to check for updates or download new updates."
msgstr ""
"Je možné, že nebudete môcť kontrolovať aktualizácie alebo sťahovať nové "
"aktualizácie."

#: ../UpdateManager/UpdateManager.py:1002
msgid "Could not initialize the package information"
msgstr "Nebolo možné inicializovať informácie o balíku"

#: ../UpdateManager/UpdateManager.py:1003
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"Počas inicializácie informácií o balíkoch sa vyskytol neriešiteľný problém.\n"
"\n"
"Prosím, nahláste to ako problém voči balíku „update-manager“ a priložte "
"nasledovnú chybovú správu:\n"

#: ../UpdateManager/UpdateManager.py:1032
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"Počas počítania aktualizácie o balíkoch sa vyskytol neriešiteľný problém.\n"
"\n"
"Prosím, nahláste to ako problém voči balíku „update-manager“ a priložte "
"nasledovnú chybovú správu:"

#: ../UpdateManager/UpdateManager.py:1056
msgid " (New install)"
msgstr " (Nová inštalácia)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:1063
#, python-format
msgid "(Size: %s)"
msgstr "(Veľkosť: %s)"

#: ../UpdateManager/UpdateManager.py:1067
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "Z verzie %(old_version)s na verziu %(new_version)s"

#: ../UpdateManager/UpdateManager.py:1071
#, python-format
msgid "Version %s"
msgstr "Verzia %s"

#: ../UpdateManager/UpdateManager.py:1104 ../do-release-upgrade:112
msgid "Release upgrade not possible right now"
msgstr "Aktualizácia vydania nie je práve teraz možná"

#: ../UpdateManager/UpdateManager.py:1105 ../do-release-upgrade:113
#, c-format, python-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Aktualizáciu vydania momentálne nie je možné vykonať. Prosím, skúste to "
"znova neskôr. Server oznámil „%s“"

#: ../UpdateManager/UpdateManager.py:1107 ../check-new-release-gtk:126
msgid "Downloading the release upgrade tool"
msgstr "Sťahuje sa nástroj na prechod na vyššiu verziu distribúcie."

#: ../UpdateManager/UpdateManager.py:1114
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>Je dostupné nové vydanie Ubuntu „%s“</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:1153
msgid "Software index is broken"
msgstr "Index softvéru je poškodený"

#: ../UpdateManager/UpdateManager.py:1154
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"V dôsledku chyby nie je možné nainštalovať alebo odstrániť žiadny program. "
"Na odstránenie tohto problému použite správcu balíkov „Synaptic“ alebo "
"spustite „sudo apt-get install -f“ v termináli."

#: ../UpdateManager/UnitySupport.py:57
msgid "Check for Updates"
msgstr "Skontrolovať aktualizácie"

#: ../UpdateManager/UnitySupport.py:66
msgid "Install All Available Updates"
msgstr "Nainštalovať všetky dostupné aktualizácie"

#: ../UpdateManagerText/UpdateManagerText.py:34
msgid "Cancel"
msgstr "Zrušiť"

#: ../UpdateManagerText/UpdateManagerText.py:37
msgid "Changelog"
msgstr "Záznam zmien"

#: ../UpdateManagerText/UpdateManagerText.py:40
msgid "Updates"
msgstr "Aktualizácie"

#: ../UpdateManagerText/UpdateManagerText.py:53
msgid "Building Updates List"
msgstr "Zostavuje sa zoznam aktualizácií"

#: ../UpdateManagerText/UpdateManagerText.py:56
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Bežnú aktualizáciu nemožno vypočítať, spustite prosím: \n"
"sudo apt-get dist-upgrade\n"
"\n"
"\n"
"Tento problém mohlo spôsobiť:\n"
"* neukončená predošlá aktualizácia\n"
"* problém s niektorým z nainštalovaného softvéru\n"
"* neoficiálne balíky softvéru, ktoré neposkytol tím Ubuntu\n"
"* bežné zmeny zatiaľ nevydanej novej verzie Ubuntu"

#: ../UpdateManagerText/UpdateManagerText.py:125
msgid "Downloading changelog"
msgstr "Sťahuje sa záznam zmien"

#: ../UpdateManager/Core/MyCache.py:147
#, python-format
msgid "Other updates (%s)"
msgstr "Ďalšie aktualizácie (%s)"

#: ../UpdateManager/Core/MyCache.py:325
msgid "This update does not come from a source that supports changelogs."
msgstr "Aktualizácia nepochádza zo zdroja, ktorý podporuje záznamy zmien."

#: ../UpdateManager/Core/MyCache.py:331 ../UpdateManager/Core/MyCache.py:359
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Nepodarilo sa stiahnuť zoznam zmien. \n"
"Skontrolujte svoje pripojenie k internetu."

#: ../UpdateManager/Core/MyCache.py:338
#, python-format
msgid ""
"Changes for the versions:\n"
"Installed version: %s\n"
"Available version: %s\n"
"\n"
msgstr ""
"Zmeny verzií:\n"
"Nainštalovaná verzia: %s\n"
"Dostupná verzia: %s\n"
"\n"

#: ../UpdateManager/Core/MyCache.py:348
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Záznam zmien neobsahuje žiadne relevantné zmeny.\n"
"\n"
"Použite prosím http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"kým nebude zoznam zmien dostupný alebo to skúste znova."

#: ../UpdateManager/Core/MyCache.py:353
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Zoznam zmien nie je práve dostupný.\n"
"\n"
"Použite, prosím, http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"pokiaľ nebudú dostupné zmeny alebo to neskôr skúste znovu."

#: ../UpdateManager/Core/UpdateList.py:51
msgid "Failed to detect distribution"
msgstr "Zlyhalo zistenie distribúcie"

#: ../UpdateManager/Core/UpdateList.py:52
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr "Chyba '%s' sa vyskytla počas kontroly systému, ktorý používate."

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Important security updates"
msgstr "Dôležité bezpečnostné aktualizácie"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Recommended updates"
msgstr "Odporúčané aktualizácie"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Proposed updates"
msgstr "Navrhované aktualizácie"

#: ../UpdateManager/Core/UpdateList.py:66
msgid "Backports"
msgstr "Backporty"

#: ../UpdateManager/Core/UpdateList.py:67
msgid "Distribution updates"
msgstr "Aktualizácie distribúcie"

#: ../UpdateManager/Core/UpdateList.py:72
msgid "Other updates"
msgstr "Ďalšie aktualizácie"

#: ../data/gtkbuilder/UpdateManager.ui.h:1
#, fuzzy
msgid "<big><b>Starting Software Updater</b></big>"
msgstr "<big><b>Spúšťa sa Správca aktualizácií</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:2
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Softvérové aktualizácie opravujú chyby, odstraňujú bezpečnostné "
"zraniteľnosti alebo poskytujú nové vlastnosti."

#: ../data/gtkbuilder/UpdateManager.ui.h:3
msgid "_Partial Upgrade"
msgstr "Č_iastočná aktualizácia"

#: ../data/gtkbuilder/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>Nie všetky aktualizácie môžu byť nainštalované</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:5
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Spustiť čiastočnú aktualizáciu, aby sa nainštalovalo čo najviac "
"aktualizácií. \n"
"\n"
"Dôvodom môže byť:\n"
" * Nedokončená predošlá aktualizácia\n"
" * Problém s niektorým nainštalovaným softvérom\n"
" * Neoficiálne balíky softvéru, ktoré nepochádzajú z Ubuntu\n"
" * Bežné zmeny vo verzii Ubuntu pred vydaním"

#: ../data/gtkbuilder/UpdateManager.ui.h:12
msgid "Chec_k"
msgstr "_Skontrolovať"

#: ../data/gtkbuilder/UpdateManager.ui.h:13
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>Musíte kontrolovať aktualizácie manuálne</big></b>\n"
"\n"
"Váš systém je nastavený aby nekontroloval aktualizácie automaticky. Toto "
"správanie môžete nastaviť v <i>Zdroje  softvéru</i> na karte "
"<i>Aktualizácie</i>."

#: ../data/gtkbuilder/UpdateManager.ui.h:16
msgid "_Hide this information in the future"
msgstr "_Túto správu už viac nezobrazovať"

#: ../data/gtkbuilder/UpdateManager.ui.h:17
msgid "Co_ntinue"
msgstr "_Pokračovať"

#: ../data/gtkbuilder/UpdateManager.ui.h:18
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Beží na batériu</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:19
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr "Váš systém je napájaný z batérie. Ste si istý, že chcete pokračovať?"

#: ../data/gtkbuilder/UpdateManager.ui.h:21
msgid "_Upgrade"
msgstr "_Aktualizovať"

#: ../data/gtkbuilder/UpdateManager.ui.h:22
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:8
msgid "Show progress of individual files"
msgstr "Zobraziť priebeh jednotlivých súborov"

#: ../data/gtkbuilder/UpdateManager.ui.h:23
#: ../data/update-manager.desktop.in.h:1
#, fuzzy
msgid "Software Updater"
msgstr "Aktualizácie softvéru"

#: ../data/gtkbuilder/UpdateManager.ui.h:24
#, fuzzy
msgid "Starting Software Updater"
msgstr "Aktualizácie softvéru"

#: ../data/gtkbuilder/UpdateManager.ui.h:25
msgid "U_pgrade"
msgstr "_Aktualizovať"

#: ../data/gtkbuilder/UpdateManager.ui.h:27
msgid "updates"
msgstr "aktualizácie"

#: ../data/gtkbuilder/UpdateManager.ui.h:28
msgid "Changes"
msgstr "Zmeny"

#: ../data/gtkbuilder/UpdateManager.ui.h:29
msgid "Description"
msgstr "Popis"

#: ../data/gtkbuilder/UpdateManager.ui.h:30
#, fuzzy
msgid "Details of updates"
msgstr "Popis aktualizácie"

#: ../data/gtkbuilder/UpdateManager.ui.h:31
msgid ""
"You are connected via roaming and may be charged for the data consumed by "
"this update."
msgstr ""
"Ste pripojený prostredníctvom roamingu a je možné, že vám bude zaúčtované "
"množstvo dát potrebných na túto aktualizáciu."

#: ../data/gtkbuilder/UpdateManager.ui.h:32
msgid ""
"You may want to wait until you’re not using a mobile broadband connection."
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:33
msgid "It’s safer to connect the computer to AC power before updating."
msgstr ""
"Je bezpečnejšie pred aktualizáciou pripojiť počítač na napájanie z "
"elektrickej siete."

#: ../data/gtkbuilder/UpdateManager.ui.h:34
msgid "_Settings..."
msgstr "Nas_tavenia..."

#: ../data/gtkbuilder/UpdateManager.ui.h:35
#, fuzzy
msgid "_Install Now"
msgstr "Nainštalovať"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>Je dostupná novšia verzia Ubuntu. Želáte si prejsť na novšiu verziu?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Don't Upgrade"
msgstr "Neaktualizovať"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Ask Me Later"
msgstr "Spýtať sa neskôr"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "Yes, Upgrade Now"
msgstr "Áno, aktualizovať teraz"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Odmietlu ste aktualizovať na novšiu verziu Ubuntu."

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
#, fuzzy
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"Môžete aktualizovať neskôr otvorením Správcu aktualizácií a kliknutím na "
"„Aktualizovať“."

#: ../data/update-manager.desktop.in.h:2
msgid "Software Updates"
msgstr "Aktualizácie softvéru"

#: ../data/update-manager.desktop.in.h:3
msgid "Show and install available updates"
msgstr "Zobraziť a nainštalovať dostupné aktualizácie"

#: ../update-manager:66 ../update-manager-text:55 ../do-release-upgrade:51
msgid "Show version and exit"
msgstr "Zobraziť verziu a skončiť"

#: ../update-manager:69
msgid "Directory that contains the data files"
msgstr "Adresár s dátovými súbormi"

#: ../update-manager:72
msgid "Check if a new Ubuntu release is available"
msgstr "Skontrolovať dostupnosť nového vydania Ubuntu"

#: ../update-manager:75 ../do-release-upgrade:54 ../check-new-release-gtk:186
msgid "Check if upgrading to the latest devel release is possible"
msgstr "Skontrolovať, či je možné aktualizovať na najnovšiu vývojársku verziu"

#: ../update-manager:79
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr ""
"Aktualizovať pomocou poslednej navrhovanej verzie aktualizátora vydania"

#: ../update-manager:86
msgid "Do not focus on map when starting"
msgstr "Pri štarte nezameriavať vstup na mapu"

#: ../update-manager:89
msgid "Try to run a dist-upgrade"
msgstr "Skúste spustiť dist-upgrade"

#: ../update-manager:92
msgid "Do not check for updates when starting"
msgstr "Nekontrolovať pri spustení aktualizácie"

#: ../update-manager:96 ../do-release-upgrade:70
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Otestovať aktualizáciu v pieskovisku aufs"

#: ../update-manager:116
msgid "Running partial upgrade"
msgstr "Prebieha čiastočná aktualizácia"

#: ../update-manager-text:59
msgid "Show description of the package instead of the changelog"
msgstr "Zobraziť popis balíka namiesto záznamu zmien"

#: ../do-release-upgrade:58 ../check-new-release-gtk:190
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Pokúste sa aktualizovať na poslednú verziu použitím aktualizátora z $distro-"
"proposed"

#: ../do-release-upgrade:62
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Beh v špeciálnom režime aktualizácie.\n"
"Momentálne sú podporované normálne aktualizácie pracovnej stanice a "
"serverových systémov."

#: ../do-release-upgrade:68
msgid "Run the specified frontend"
msgstr "Spustiť uvedený frontend."

#: ../do-release-upgrade:73
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Skontrolovať iba ak je dostupné nové vydanie distribúcie a oznámiť výsledok "
"návratovou hodnotou"

#: ../do-release-upgrade:87
msgid "Checking for a new Ubuntu release"
msgstr "Kontroluje sa dostupnosť nového vydania Ubuntu"

#: ../do-release-upgrade:101
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Informácie o aktualizáciách nájdete na:\n"
"%(url)s\n"

#: ../do-release-upgrade:107
msgid "No new release found"
msgstr "Žiadne nové vydanie nebolo nájdené"

#: ../do-release-upgrade:119
#, c-format
msgid "New release '%s' available."
msgstr "Je dostupné nové vydanie „%s“."

#: ../do-release-upgrade:120
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Ak naň chcete aktualizovať systém, spustite „do-release-upgrade“."

#: ../check-new-release-gtk:101
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Dostupná nová verzia Ubuntu %(version)s"

#: ../check-new-release-gtk:143
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Odmietli ste aktualizovať na Ubuntu %s"

#: ../check-new-release-gtk:196
msgid "Add debug output"
msgstr "Pridať ladiaci výstup"

#: ../ubuntu-support-status:91
msgid "Show unsupported packages on this machine"
msgstr "Zobraziť nepodporované balíky na tomto počítači"

#: ../ubuntu-support-status:94
msgid "Show supported packages on this machine"
msgstr "Zobraziť podporované balíky na tomto počítači"

#: ../ubuntu-support-status:97
msgid "Show all packages with their status"
msgstr "Zobraziť všetky balíky a ich stav"

#: ../ubuntu-support-status:100
msgid "Show all packages in a list"
msgstr "Zobraziť všetky balíky v zozname"

#: ../ubuntu-support-status:142
#, c-format
msgid "Support status summary of '%s':"
msgstr "Zhrnutie stavu podpory „%s“"

#: ../ubuntu-support-status:145
msgid "You have %(num)s packages (%(percent).1f%%) supported until %(time)s"
msgstr "Máte %(num)s balíkov (%(percent).1f%%) podporovaných do %(time)s"

#: ../ubuntu-support-status:151
msgid ""
"You have %(num)s packages (%(percent).1f%%) that can not/no-longer be "
"downloaded"
msgstr "Máte %(num)s balíkov (%(percent).1f%%), ktoré (už) nemožno stiahnuť"

#: ../ubuntu-support-status:154
msgid "You have %(num)s packages (%(percent).1f%%) that are unsupported"
msgstr "Máte %(num)s balíkov (%(percent).1f%%)m ktoré sú nepodporované"

#: ../ubuntu-support-status:162
msgid ""
"Run with --show-unsupported, --show-supported or --show-all to see more "
"details"
msgstr ""
"Ďalšie podrobnosti uvidíte po spustení s voľbami --show-unsupported, --show-"
"supported alebo --show-all"

#: ../ubuntu-support-status:166
msgid "No longer downloadable:"
msgstr "Už sa viac nedá stiahnuť:"

#: ../ubuntu-support-status:169
msgid "Unsupported: "
msgstr "Nepodporované: "

#: ../ubuntu-support-status:174
#, c-format
msgid "Supported until %s:"
msgstr "Podporované do %s:"

#: ../ubuntu-support-status:183
msgid "Unsupported"
msgstr "Nepodporované"

#. Why do we use %s here instead of $strings or {} format placeholders?
#. It's because we don't want to break existing translations.
#: ../janitor/plugincore/exceptions.py:42
#, python-format
msgid "Unimplemented method: %s"
msgstr "neimplementovaná metóda: %s"

#: ../janitor/plugincore/core/file_cruft.py:41
msgid "A file on disk"
msgstr "Súbor na disku"

#: ../janitor/plugincore/core/missing_package_cruft.py:39
msgid "Install missing package."
msgstr "Nainštalovať chýbajúci balík."

#. 2012-06-08 BAW: i18n string; don't use {} or PEP 292.
#: ../janitor/plugincore/core/missing_package_cruft.py:49
#, python-format
msgid "Package %s should be installed."
msgstr "Balík %s by mal byť nainštalovaný."

#: ../janitor/plugincore/core/package_cruft.py:49
msgid ".deb package"
msgstr "balík .deb"

#: ../janitor/plugincore/plugins/langpack_manual_plugin.py:46
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "%s je potrebné označiť ako manuálne nainštalovaný."

#: ../janitor/plugincore/plugins/kdelibs4to5_plugin.py:49
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""
"Ak je počas aktualizácie nainštalovaný kdelibs4-dev je potrebné nainštalovať "
"kdelibs5-dev. Podrobnosti pozri v hlásení chyby #279621 na bugs.launchpad.net"

#: ../janitor/plugincore/plugins/dpkg_status_plugin.py:44
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "%i zastaraných záznamov v stavovom súbore"

#: ../janitor/plugincore/plugins/dpkg_status_plugin.py:47
msgid "Obsolete entries in dpkg status"
msgstr "Zastarané položky v stave dpkg"

#. pragma: no cover
#: ../janitor/plugincore/plugins/dpkg_status_plugin.py:50
msgid "Obsolete dpkg status entries"
msgstr "Zastarané stavové položky dpkg"

#: ../janitor/plugincore/plugins/remove_lilo_plugin.py:42
msgid "Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"Odstrániť lilo, pretože je nainštalovaný grub. (Podrovnosti v hlásení chyby "
"#314004)"

#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore so a bug reporting process is being started."
#~ msgstr ""
#~ "Po aktualizácii informácií o balíkoch už nie je možné nájsť nevyhnutný "
#~ "balík „%s“preto sa spúšťa proces hlásenia chyby ."

#~ msgid "<b><big>Upgrading Ubuntu to version 12.04</big></b>"
#~ msgstr "<b><big>Aktualizuje sa Ubuntu na verziu 12.04</big></b>"

#~ msgid "%(count)s update has been selected."
#~ msgid_plural "%(count)s updates have been selected."
#~ msgstr[0] "Bolo vybraných %(count)s aktualizácií."
#~ msgstr[1] "Bola vybraná %(count)s aktualizácia."
#~ msgstr[2] "Boli vybrané %(count)s aktualizácie."

#~ msgid "%(count_str)s %(download_str)s"
#~ msgstr "%(count_str)s %(download_str)s"

#~ msgid "Welcome to Ubuntu"
#~ msgstr "Vitajte v Ubuntu"

#~ msgid ""
#~ "These software updates have been issued since this version of Ubuntu was "
#~ "released."
#~ msgstr ""
#~ "Tieto aktualizácie softvéru boli vydané od vydania tejto verzie Ubuntu."

#~ msgid "Software updates are available for this computer."
#~ msgstr "Pre tento počítač sú k dispozícii aktualizácie softvéru."

#~ msgid "Update Manager"
#~ msgstr "Správca aktualizácií"

#~ msgid "Starting Update Manager"
#~ msgstr "Spúšťa sa Správca aktualizácií"

#~ msgid "You are connected via a wireless modem."
#~ msgstr "Ste pripojený pomocou bezdrôtového modemu."

#~ msgid "_Install Updates"
#~ msgstr "_Nainštalovať aktualizácie"

#~ msgid "Checking for a new ubuntu release"
#~ msgstr "Kontroluje sa, či existuje novšia verzia distribúcie Ubuntu"

#~ msgid "Your system is up-to-date"
#~ msgstr "Váš systém je aktuálny"

#~ msgid "Software updates are available for this computer"
#~ msgstr "Pre tento počítač sú dostupné aktualizácie"

#~ msgid ""
#~ "If you don't want to install them now, choose \"Update Manager\" from the "
#~ "Administration menu later."
#~ msgstr ""
#~ "Ak si neželáte ich inštalovať teraz, zvoľte neskôr „Správca aktualizácií“ "
#~ "z ponuky Správa."

#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written "
#~ "to '%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a systemdir from now until the next reboot are "
#~ "permanent."
#~ msgstr ""
#~ "Táto aktualizácia beží v režime pieskoviska (v testovacom režime). "
#~ "Akékoľvek zmeny budú zapísané do „%s“ a stratia sa pri ďalšom reštarte.\n"
#~ "\n"
#~ "*Žiadne* zmeny zapísané do systémového priečinka odteraz do ďalšieho "
#~ "reštartu za nezachovajú."

#~ msgid ""
#~ "Fetching and installing the upgrade can take several hours. Once the "
#~ "download has finished, the process cannot be cancelled."
#~ msgstr ""
#~ "Sťahovanie a inštalácia aktualizácie môže trvať niekoľko hodín a nemôže "
#~ "byť neskôr prerušená."

#~ msgid "There are no updates to install"
#~ msgstr "Žiadne aktualizácie nečakajú na inštaláciu"

#~ msgid "The update has already been downloaded, but not installed"
#~ msgid_plural "The updates have already been downloaded, but not installed"
#~ msgstr[0] "Aktualizácie boli stiahnuté, ale nie nainštalované."
#~ msgstr[1] "Aktualizácia bola stiahnutá, ale nie nainštalovaná."
#~ msgstr[2] "Aktualizácie boli stiahnuté, ale nie nainštalované."

#~ msgid ""
#~ "You will not get any further security fixes or critical updates. Please "
#~ "upgrade to a later version of Ubuntu Linux."
#~ msgstr ""
#~ "Nedostanete žiadne ďalšie bezpečnostné aktualizácie ani dôležité "
#~ "aktualizácie. Prosím, aktualizujte na novšiu verziu Ubuntu Linux."

#~ msgid ""
#~ "If you don't want to install them now, choose \"Update Manager\" from "
#~ "Applications later."
#~ msgstr ""
#~ "Ak ich nechcete inštalovať teraz, zvoľte „Správca aktualizácií“ neskôr z "
#~ "menu Aplikácie."

#~ msgid ""
#~ "The system was unable to get the prerequisites for the upgrade. The "
#~ "upgrade will abort now and restore the original system state.\n"
#~ "\n"
#~ "Please report this as a bug using the command 'ubuntu-bug update-manager' "
#~ "in a terminal and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Systému sa nepodarilo získať predpoklady pre aktualizáciu. Aktualizácia "
#~ "sa teraz preruší a obnoví sa pôvodný stav systému.\n"
#~ "\n"
#~ "Prosím, nahláste to ako chybu príkazom „ubuntu-bug update-manager“ v "
#~ "termináli a priložte do hlásenia súbory z /var/log/dist-upgrade/"

#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore.\n"
#~ "This indicates a serious error, please report this bug using the command "
#~ "'ubuntu-bug update-manager' in a terminal and include the files in /var/"
#~ "log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "Potom ako boli aktualizované vaše informácie o balíkoch už nie je možné "
#~ "nájsť nevyhnutný balík „%s“.\n"
#~ "Prosím, nahláste to ako chybu príkazom „ubuntu-bug update-manager“ v "
#~ "termináli a priložte do hlásenia súbory z /var/log/dist-upgrade/"

#~ msgid ""
#~ "\n"
#~ "\n"
#~ "Please report this bug using the command 'ubuntu-bug update-manager' in a "
#~ "terminal and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report.\n"
#~ "%s"
#~ msgstr ""
#~ "\n"
#~ "\n"
#~ "Prosím, nahláste to ako chybu príkazom „ubuntu-bug update-manager“ v "
#~ "termináli a priložte do hlásenia súbory z /var/log/dist-upgrade/\n"
#~ "%s"

#~ msgid ""
#~ "Preparing the system for the upgrade failed. Please report this using the "
#~ "command 'ubuntu-bug update-manager' in a terminal and include the files "
#~ "in /var/log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "Príprava systému na aktualizáciu zlyhala. Prosím, nahláste to ako chybu "
#~ "príkazom „ubuntu-bug update-manager“ v termináli a priložte do hlásenia "
#~ "súbory z /var/log/dist-upgrade/"

#~ msgid ""
#~ "Upgrading the repository information resulted in a invalid file. Please "
#~ "report this as a bug using the command 'ubuntu-bug update-manager' in a "
#~ "terminal."
#~ msgstr ""
#~ "Aktualizácia informácií zdroja softvéru mala za výsledok neplatný súbor. "
#~ "Prosím, nahláste to ako chybu príkazom „ubuntu-bug update-manager“ v "
#~ "termináli."

#~ msgid "Your graphics hardware may not be fully supported in Ubuntu 11.04."
#~ msgstr "Ubuntu 11.04 nemusí plne podporovať váš grafický hardvér."

#~ msgid ""
#~ "The support in Ubuntu 11.04 for your intel graphics hardware is limited "
#~ "and you may encounter problems after the upgrade. Do you want to continue "
#~ "with the upgrade?"
#~ msgstr ""
#~ "Podpora vášho grafického hardvéru Intel v Ubuntu 11.04 je obmedzená a po "
#~ "aktualizácii môžete naraziť na problémy. Chcete pokračovať v aktualizácii?"

#~ msgid "0 kB"
#~ msgstr "0 kB"

#~ msgid "1 kB"
#~ msgstr "1 kB"

#~ msgid "%.0f kB"
#~ msgstr "%.0f kB"

#~ msgid ""
#~ "These software updates have been issued since this version of Ubuntu was "
#~ "released. If you don't want to install them now, choose \"Update Manager"
#~ "\" from Applications later."
#~ msgstr ""
#~ "Tieto aktualizácie softvéru boli vydané od uvoľnenia tejto verzie Ubuntu. "
#~ "Ak ich nechcete nainštalovať teraz, zvoľte neskôr „Správca aktualizácií“ "
#~ "z Aplikácií."

#~ msgid ""
#~ "These software updates have been issued since this version of Ubuntu was "
#~ "released. If you don't want to install them now, choose \"Update Manager"
#~ "\" from the Administration Menu later."
#~ msgstr ""
#~ "Tieto aktualizácie softvéru boli vydané od uvoľnenia tejto verzie Ubuntu. "
#~ "Ak ich nechcete nainštalovať teraz, zvoľte neskôr „Správca aktualizácií“ "
#~ "z menu Správa."
